pom.xml

```diff
@@ -93,6 +66,38 @@
           <target>1.7</target>
         </configuration>
       </plugin>
+      <!--plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.1.1</version>
+        <executions>
+          <execution>
+            <id>unpack</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>com.github.ua-parser</groupId>
+                  <artifactId>uap-core</artifactId>
+                  <outputDirectory>${project.build.directory}/classes</outputDirectory>
+                  <includes>**/*.yaml</includes>
+                </artifactItem>
+                <artifactItem>
+                  <groupId>com.github.ua-parser</groupId>
+                  <artifactId>uap-core</artifactId>
+                  <classifier>tests</classifier>
+                  <type>test-jar</type>
+                  <outputDirectory>${project.build.directory}/test-classes</outputDirectory>
+                  <includes>**/*.yaml</includes>
+                </artifactItem>
+              </artifactItems>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin-->
```

Comment:
why is this all commented out?

Reply:
Sorry, commented it out to see if unpacking was necessary. It wasn't so I will delete it and update the pull request.